Makefile

```diff
@@ -39,20 +37,26 @@ build-builder-bionic: build-linux-packages build-sample-root
 build-linux-buildpacks: build-buildpacks-alpine build-buildpacks-bionic
 
 build-buildpacks-alpine: build-sample-root
+	@echo "> Starting local registry to store alpine builder (when builder contains extensions it must exist in a registry so that builds can use --pull-policy=always and we don't want to override the locally built builder)"
+	docker run -d --rm -p 5000:5000 registry:2
+	sleep 2
+	docker tag cnbs/sample-builder:alpine localhost:5000/cnbs/sample-builder:alpine
+	docker push localhost:5000/cnbs/sample-builder:alpine
+
```

Comment:
Is this a good use-case for `docker/podman-compose`?  We could spin up `registry:2` in one container, and the application builds in another?  This would avoid the need for `--network=host`.

Reply:
Maybe... do you think the benefit is worth the additional overhead?

Reply:
No. Let's go with it the way it is.